openspec/changes/archive/2026-07-18-add-eve-admin-mount-global-panel/tasks.md

```
@@ -0,0 +1,29 @@
+## 1. Mount the isolated runtime
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)